assets/sass/admin.scss

```diff
@@ -143,6 +143,7 @@
 @import "components/global/googlesitekit-user-menu";
 @import "components/global/googlesitekit-view-only-menu";
 @import "components/global/googlesitekit-warning-notice";
+@import "components/global/googlesitekit-badge-with-tooltip";
```

Comment:
The imports in this file generally follow an alphabetically ordered manner. Let's try to maintain that.